File: Hard/2025-10-30-1526-Minimum-Number-of-Increments-on-Subarrays-to-Form-a-Target-Array/solution.py
class Solution:
    def minOperations(self, target: List[int]) -> int:
        operations = 0
        for i in range(1, len(target)):
            if target[i] > target[i - 1]:
                operations += target[i] - target[i - 1]
        operations += target[0]  # Add the first element's value
        return operations

File: Hard/2025-10-30-1526-Minimum-Number-of-Increments-on-Subarrays-to-Form-a-Target-Array/solution.java
class Solution {
    public int minNumberOfOperations(int[] target) {
        int operations = 0;
        for (int i = 1; i < target.length; i++) {
            if (target[i] > target[i - 1]) {
                operations += target[i] - target[i - 1];
            }
        }
        operations += target[0]; // Increment to reach the first element
        return operations;
    }
}